apps/sms/services/js/message_manager.js

```diff
@@ -396,26 +500,42 @@ var MessageManager = {
     });
   },
 
+  /**
+   * Method used mark a thread (all messages in it) as read.
```

Comment:
@martiansideofthemoon  Method used to mark a thread as read or unread depending on it's value as true or false respectively.

(This is what markThreadRead does, if you can frame/modify this string better, it would be nice)